lib/Git/ProcessBuilder.php

```diff
@@ -27,7 +27,7 @@ public function __construct(array $arguments = array())
     {
         array_unshift($arguments, 'git');
         $gitDir = \Mesamatrix::path(\Mesamatrix::$config->getValue("info", "private_dir"))."/";
-        $gitDir .= \Mesamatrix::$config->getValue("git", "dir");
+        $gitDir .= \Mesamatrix::$config->getValue("git", "dir", "mesa.git");
```

Comment:
I just noticed that these values could do with a default. It's unrelated to the actual PR :laughing: